ci/release/update-version.sh

```diff
@@ -49,6 +49,7 @@ DEPENDENCIES=(
   dask-cuda
   dask-cudf
   kvikio
+  libcudf
   libkvikio
   librmm
   rapids-dask-dependency
```

Comment:
See #16588, we missed adding pylibcudf. Let's fix it here.
```suggestion
  pylibcudf
  rapids-dask-dependency
```

Reply:
sure, added a `fixes #16588` to the PR description.